build.sbt

```diff
@@ -94,5 +94,7 @@ ThisBuild / githubWorkflowAddedJobs += WorkflowJob(
         "publish_dir" -> "./target/website",
         "github_token" -> "${{ secrets.GITHUB_TOKEN }}"
       )
-    )
+    ),
+  scalas = List(Scala213),
+  javas = githubWorkflowJavaVersions.value.toList
```

Comment:
man couldn't decide if he prefers Seq or List so there's both in the API :D @djspiewak

Reply:
Seq for sbt (where you get the appenders), and List for everything else!

Reply:
you don't get appenders for List? 🤔

Reply:
```scala
val x = settingKey[List[String]]("foo")
x := List("hello")
x += "world"
```

seems to work:

```
sbt:root> show x
[info] * hello
[info] * world
```

Reply:
Hmm. That *didn't* work when I was building the plugin. I wonder if it was later fixed?

Reply:
Possible :) although https://github.com/sbt/sbt/blame/develop/main-settings/src/main/scala/sbt/Append.scala hasn't been modified in years